requirements/doc.txt

```diff
@@ -4,49 +4,36 @@
 #
 #    make upgrade
 #
-alabaster==0.7.12         # via sphinx
```

Comment:
A lot of dependencies went missing in this file, please add `--rebuild` to the pip-compile commands in `Makefile` to avoid any caching of bad dependency data.